### PR TITLE
Add some more features to the cpuid table.

### PR DESCRIPTION
**Pull request by incase**
Information taken from https://www.lowlevel.eu/wiki/CPUID
Only added flags I actually have some interest in: long mode (64 bit support), NX , huge page support, ...
Some of these could probably be inferred from other flags (or their combination) but it seems cleaner to just pull them from the cpuid functions directly.

### CODE REVIEW

**Comment by muffins** [comment]
ok to test

**Comment by osqueryer** [comment]
:-1: The commit 13c958163246e9a17f560a1c2595625d6cdfa1b1 (Job results: [2692](https://jenkins.osquery.io/job/osqueryPullRequestOSX/2692/)) failed one or more tests (**macOS/OS X**).

**Comment by osqueryer** [comment]
:-1: The commit 13c958163246e9a17f560a1c2595625d6cdfa1b1 (Job results: [2693](https://jenkins.osquery.io/job/osqueryPullRequestOSX/2693/)) failed one or more tests (**macOS/OS X**).

**Comment by osqueryer** [comment]
:-1: The commit 13c958163246e9a17f560a1c2595625d6cdfa1b1 (Job results: [6624](https://jenkins.osquery.io/job/osqueryPullRequestLinux/6624/)) failed one or more tests (**Linux**).

**Comment by osqueryer** [comment]
:-1: The commit 13c958163246e9a17f560a1c2595625d6cdfa1b1 (Job results: [6625](https://jenkins.osquery.io/job/osqueryPullRequestLinux/6625/)) failed one or more tests (**Linux**).

**Comment by osqueryer** [comment]
:-1: The commit 13c958163246e9a17f560a1c2595625d6cdfa1b1 (Job results: [1131](https://jenkins.osquery.io/job/osqueryPullRequestFreeBSD/1131/)) failed one or more tests (**FreeBSD**).

**Comment by osqueryer** [comment]
:-1: The commit 13c958163246e9a17f560a1c2595625d6cdfa1b1 (Job results: [1132](https://jenkins.osquery.io/job/osqueryPullRequestFreeBSD/1132/)) failed one or more tests (**FreeBSD**).

**Comment by osqueryer** [comment]
:-1: The commit 13c958163246e9a17f560a1c2595625d6cdfa1b1 (Job results: [3520](https://jenkins.osquery.io/job/osqueryPullRequestWindows/3520/)) failed one or more tests (**Windows**).

**Comment by osqueryer** [comment]
:-1: The commit 13c958163246e9a17f560a1c2595625d6cdfa1b1 (Job results: [3521](https://jenkins.osquery.io/job/osqueryPullRequestWindows/3521/)) failed one or more tests (**Windows**).

**Comment by osqueryer** [comment]
:no_good: The commit e816c16862c1ba39e85e8f0056fb5acaf06e4af6 (Job results: [3234](https://jenkins.osquery.io/job/osqueryPullRequestAudit/3234/)) failed the code audit or documentation test.

**Comment by incase** [comment]
Ohoh, I see what went wrong.
New commit added to fix it.

**Comment by osqueryer** [comment]
:-1: The commit e816c16862c1ba39e85e8f0056fb5acaf06e4af6 (Job results: [3575](https://jenkins.osquery.io/job/osqueryPullRequestWindows/3575/)) failed one or more tests (**Windows**).

**Comment by osqueryer** [comment]
:-1: The commit e816c16862c1ba39e85e8f0056fb5acaf06e4af6 (Job results: [3576](https://jenkins.osquery.io/job/osqueryPullRequestWindows/3576/)) failed one or more tests (**Windows**).

**Comment by osqueryer** [comment]
:-1: The commit e816c16862c1ba39e85e8f0056fb5acaf06e4af6 (Job results: [6686](https://jenkins.osquery.io/job/osqueryPullRequestLinux/6686/)) failed one or more tests (**Linux**).

**Comment by osqueryer** [comment]
:-1: The commit e816c16862c1ba39e85e8f0056fb5acaf06e4af6 (Job results: [6687](https://jenkins.osquery.io/job/osqueryPullRequestLinux/6687/)) failed one or more tests (**Linux**).

**Comment by incase** [comment]
Could someone help me figuring out why the checks fail? Looking at https://jenkins.osquery.io/job/osqueryPullRequestWindows/3576/console for example, it looks as if the runner is unavailable and thus the build is never actually attempted?

Or for the Code Audit one:
New build name is '43853f7dbce317f6db34e3bc00016d781a5cca7e'
Triggering osqueryPullRequestAudit » aws-ubuntu-16.04
osqueryPullRequestAudit » aws-ubuntu-16.04 completed with result FAILURE
Set build name.
New build name is '43853f7dbce317f6db34e3bc00016d781a5cca7e'
Setting status of e816c16862c1ba39e85e8f0056fb5acaf06e4af6 to FAILURE with url https://jenkins.osquery.io/job/osqueryPullRequestAudit/3234/ and message: 'Build finished. '
Using context: Code Audit
:no_good: The commit e816c16862c1ba39e85e8f0056fb5acaf06e4af6 (Job results: [3234](https://jenkins.osquery.io/job/osqueryPullRequestAudit/3234/)) failed the code audit or documentation test.

Note that building and testing with my patches on Ubuntu 16.04 works ("make deps; make; make test" all return no errors). I don't have a Windows system to test on, but the I couldn't see anything useful in the logs of that failure either.

**Comment by muffins** [comment]
ok to test

**Comment by muffins** [comment]
@incase sorry about that :) Our Jenkins CI had some hiccups over the past few days. The Windows might still be broken, but if everything else passes we'll call this good and land.

**Comment by osqueryer** [comment]
:no_good: The commit e816c16862c1ba39e85e8f0056fb5acaf06e4af6 (Job results: [3287](https://jenkins.osquery.io/job/osqueryPullRequestAudit/3287/)) failed the code audit or documentation test.

**Comment by osqueryer** [comment]
:-1: The commit e816c16862c1ba39e85e8f0056fb5acaf06e4af6 (Job results: [2801](https://jenkins.osquery.io/job/osqueryPullRequestOSX/2801/)) failed one or more tests (**macOS/OS X**).

**Comment by osqueryer** [comment]
:-1: The commit e816c16862c1ba39e85e8f0056fb5acaf06e4af6 (Job results: [3668](https://jenkins.osquery.io/job/osqueryPullRequestWindows/3668/)) failed one or more tests (**Windows**).

**Comment by osqueryer** [comment]
:-1: The commit e816c16862c1ba39e85e8f0056fb5acaf06e4af6 (Job results: [6773](https://jenkins.osquery.io/job/osqueryPullRequestLinux/6773/)) failed one or more tests (**Linux**).

**Comment by osqueryer** [comment]
:no_good: The commit 175133c0945b5c25f9e99f6e385481888bab3d9f (Job results: [3569](https://jenkins.osquery.io/job/osqueryPullRequestAudit/3569/)) failed the code audit or documentation test.

**Comment by osqueryer** [comment]
:-1: The commit 175133c0945b5c25f9e99f6e385481888bab3d9f (Job results: [4000](https://jenkins.osquery.io/job/osqueryPullRequestWindows/4000/)) failed one or more tests (**Windows**).

**Comment by muffins** [comment]
@incase looks like we just need to remove that one extra newline and we'll be good to go to land this, thanks!

**Comment by fmanco** [comment]
@incase I'll update your PR with the fix so that we can get this merged.

File: osquery/tables/system/cpuid.cpp [hunk 1]
```diff
@@ -69,10 +69,16 @@ std::map<size_t, std::vector<FeatureDef>> kCPUFeatures{
      }},
     {0x80000001,
      {
+         FEATURE("lahfsahf", "ecx", 0),
          FEATURE("svm", "ecx", 2),
+         FEATURE("prefetchw", "ecx", 8),
```
Comment (fmanco) [inline osquery/tables/system/cpuid.cpp]: Shouldn't this be called `3dnowprefetch`?

**Comment by fmanco** [comment]
jenkins test audit

**Comment by fmanco** [comment]
jenkins test linux

**Comment by incase** [comment]
Sorry for the lack of responses. I had been away from keyboards for a while.
Thanks for following through with the fixes and merging. Your help with that is greatly appreciated.

**Comment by fmanco** [comment]
You're welcome @incase. Thanks for submitting the PR in the first place.

**Pull request merged**